prime-dotnet-webapi/Services/Rules/AutomaticAdjudicationRules.cs

```diff
@@ -280,6 +282,16 @@ public override async Task<bool> ProcessRule(Enrollee enrollee)
                     enrollee.AddReasonToCurrentStatus(StatusReasonType.PaperEnrolmentMismatch, $"User-Provided GPID: {potentialPaperEnrolleeGpid}");
                     return false;
                 }
+
+                // First enrolment: check if the related paper enrolment is flagged for AlwaysManual
+                // If so, mark BCSC enrolment as AlwaysManual too and send to manual enrolment
+                if (await _enrolleePaperSubmissionService.IsAlwaysManualEnrolment(paperEnrolleeMatchId))
```

Comment:
you already have the matching `Enrollee` object above in the line `var paperEnrollees = ...`, which has the flag on it already. You don't really need to punch out again.

Reply:
Oh that's actually true,. Will remove the redundant check

Reply:
Updated